tests/acceptance/content.feature

```diff
@@ -0,0 +1,37 @@
+Feature: content
+  In order to manage content article in the web
+  As an owner
+  I need to create modify delete and un public content article
+
+  Background:
+    Given There is an administrator link
+    When Login into Joomla administrator with username "admin" and password "admin"
+    Then I see administrator dashboard
+
+  Scenario: Feature an Article
+    Given there is a content article with title "My first Joomla Article"
+    When I feature the content with title "My first Joomla Article"
+    Then I should see the published article "My first Joomla Article" in the joomla home page
+
+    Scenario: Modify an article
+      Given I select the content article with title "My first Joomla Article"
+      And change the title "My Joomla Article"
+      When I try to post "My Joomla Article"
+      Then I should see the "Article successfully saved"
+      And published article "My Joomla Article" in the joomla home page
+
+  Scenario: Unpublish an article
+      Given I have article with name "My Joomla Article"
+      When I un publish "My Joomla Article"
+      Then I can not see the article"My Joomla Article" in the joomla home page
+
+  Scenario: Remove an article
```

Comment:
we will need to "trash":

![screen shot 2016-05-12 at 16 08 28](https://cloud.githubusercontent.com/assets/1375475/15217427/d2bab89a-185b-11e6-88e3-02a4b383dfab.png)

And then we need also to empty the trash when the article is already in the trash:

![capture 2016-05-04 at 16 10 01](https://cloud.githubusercontent.com/assets/1375475/15217492/0a531b4e-185c-11e6-93e5-22d9eaba47bb.png)